### PR TITLE
Flow Diagram

### PR DESCRIPTION
**Pull request by andresuribe87**
Addresses #31 

The flow diagram has the additional step of the holder retrieving the status VC.

I am not convinced this should be part of the spec because the credentialStatus is an extension point, and different types could work in different ways.

### CODE REVIEW

File: draft-terbu-sd-jwt-vc.md
Comment (awoie) [inline draft-terbu-sd-jwt-vc.md:180]: Thanks for the PR. can you change the terms used for the labels of the lines? Let's avoid SD-JWT and use Verifiable Credentials and Verifiable Presentations instead.

**Review by awoie — CHANGES_REQUESTED** [review]
Thank you. LGTM, just needs to change the terms of the labels.

File: draft-terbu-sd-jwt-vc.md
Comment (danielfett) [inline draft-terbu-sd-jwt-vc.md:177]: We did not introduce the status provider yet. If we want to have it here (can make sense), let's introduce it somewhere first.

Comment (andresuribe87) [reply draft-terbu-sd-jwt-vc.md:181]: Added it to the terminology section.

File: draft-terbu-sd-jwt-vc.md
Comment (andresuribe87) [reply draft-terbu-sd-jwt-vc.md]: Done in [cad5733](https://github.com/vcstuff/draft-terbu-sd-jwt-vc/pull/55/commits/cad5733884ac2c61748350a314965f95e8a5d749)

File: draft-terbu-sd-jwt-vc.md
Comment (awoie) [inline draft-terbu-sd-jwt-vc.md:140]:
```suggestion
: An entity that provides status information (e.g. revocation) about a Verifiable Credential.
```

**Review by awoie — APPROVED** [review]
Thank you very much! LGTM.

**Review by awoie — CHANGES_REQUESTED** [review]
Can you pls fix the following issues:

```
/github/workspace/draft-terbu-sd-jwt-vc.xml(165): Warning: Too long line found (L221), 3 characters longer than 72 characters: 
              +------------+                                 +------------+
/github/workspace/draft-terbu-sd-jwt-vc.xml(165): Warning: Too long line found (L222), 3 characters longer than 72 characters: 
              |            |                                 |   Status   |
/github/workspace/draft-terbu-sd-jwt-vc.xml(165): Warning: Too long line found (L223), 3 characters longer than 72 characters: 
              |   Holder   |-------- optionally ------------>|  Provider  |
/github/workspace/draft-terbu-sd-jwt-vc.xml(165): Warning: Too long line found (L2[24](https://github.com/vcstuff/draft-terbu-sd-jwt-vc/actions/runs/5055609076/jobs/9072030782?pr=55#step:9:25)), 3 characters longer than 72 characters: 
              |            |       retrieves status          |            |
/github/workspace/draft-terbu-sd-jwt-vc.xml(165): Warning: Too long line found (L2[25](https://github.com/vcstuff/draft-terbu-sd-jwt-vc/actions/runs/5055609076/jobs/9072030782?pr=55#step:9:26)), 3 characters longer than 72 characters: 
              +------------+             VC                  +------------+
/github/workspace/draft-terbu-sd-jwt-vc.xml(748): Warning: Too long line found (L854), 3 characters longer than 72 characters: 
  vc.credentialSubject = insert_claims_into_credential_subject(vc, payload)
lib/trace.sh draft-terbu-sd-jwt-vc.html -s xml2rfc-html xml2rfc -q -s 'Setting consensus="true" for IETF STD document' --rfc-base-url https://www.rfc-editor.org/rfc/ --id-base-url https://datatracker.ietf.org/doc/html/ --cache=/github/workspace/.refcache --css=lib/v3.css --metadata-js-url=/dev/null draft-terbu-sd-jwt-vc.xml -o draft-terbu-sd-jwt-vc.html --html
181:           +------------+                                 +------------+ 
185:           +------------+             VC                  +------------+ 
*** Run 'make fix-lint' to automatically fix some errors
draft-terbu-sd-jwt-vc.md contains trailing whitespace
make[1]: *** [lib/main.mk:201: lint-whitespace] Error 1
make[1]: Target 'lint' not remade because of errors.
rm draft-terbu-sd-jwt-vc.xml
make: *** [lib/main.mk:16: all] Error 2
```

**Comment by danielfett** [comment]
AFAIK, only the trailing whitespace prevents compiling - I don't think the long lines do, too.

**Review by awoie — APPROVED** [review]
lgtm now. thank you!

**Comment by andresuribe87** [comment]
@awoie is it expected for the github actions check to fail? Not sure how to debug that one.

**Comment by awoie** [comment]
> @awoie is it expected for the github actions check to fail? Not sure how to debug that one.

I think this issue is due to another PR I just merged. @danielfett might be able to fix it. I have to roll back the previous PR.

**Comment by awoie** [comment]
> @andresuribe87 can you rebase main?

I mean sync from vcstuff:main and then rebase because you will need #76

**Comment by andresuribe87** [comment]
@awoie cool, waiting for workflow approval now.

**Pull request merged**
